### PR TITLE
Sam Wolf's bug fix

### PR DESCRIPTION
**Pull request by flowmas**

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -238,7 +238,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
-    // addOption("theme name", "linear-gradient(blue, red)", "red", "red");
+    addOption("Wolf theme", "linear-gradient(blue, orange)", "blue", "white");
```
Comment (Erica-Fer) [inline calculator.js:240]: I have bad tastes in colors... here are my bad taste suggestions!

```suggestion
    addOption("Wolf theme", "linear-gradient(green, pink)", "blue", "white");
```

File: calculator.js [hunk 2]
```diff
@@ -238,7 +238,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
```
Comment (Erica-Fer) [inline calculator.js]: Don't need this line anymore

```suggestion
```

**Review by Erica-Fer — CHANGES_REQUESTED** [review]
Overall, looks good!

**Review by Erica-Fer — APPROVED** [review]

**Review by etcadinfinitum — COMMENTED** [review]
Looks fine to me!

Consider adding a description to the PR body in the future; it's helpful in many situations.

Pro Tip: under "Related Issues", you can [use GitHub keywords to automatically close issues](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) if you like; this can be done in the PR description or any of the commits on the feature branch (title or body).

**Pull request merged**